standard/standard.md

```diff
@@ -0,0 +1,80 @@
+
+
+## What ALS Is
+
+ALS is a schema that defines things like the names of various lattice element types, how to organize lattice
+elements into lines which beams of particles or X-rays can move through, etc. 
+
+## What ALS Is Not
+
+ALS does not define any particular grammer to implemement the ALS schema. Rather, there are associated
+language specific standards that define grammers for YAML, JSON, Python, etc. Along with these
+associated standards, there are packages that implement translation between lattice files and a representational
+internal format defined by the package.
+
+ALS does not define how particles are to be tracked through a lattice. ALS is for describing machines and
+not for defining how to simulate particle motion. 
+
+## Conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
+
+All `keywords` in this standard are case-sensitive.
+
+## Lattice Elements
+
+The basic building block used to describe an accelerator is the lattice \vn{element}. Typically,
+a lattice element is something physical like a bending magnet or an electrostatic
+quadrupole, or a diffracting crystal. A lattice element may define a region in space 
+distinguished by the presense of (possibly time-varying) electromagnetic fields,
+materials, apertures and other possible engineered structures. However, lattice elements
+are not restricted to being something physical and may, for example, just mark a particular point in space
+(EG: **Marker** elements), or may designate where beam lines intersect (**Fork** elements).
+By convention, element names in ALS will be upper camel case.
+
+
+## Lattice Branches
+
+A lattice **Branch** is essentially an ordered array of lattice elements.
+In the simplist case, a program can track through the elements one element at a time.
+However, lattice elements may overlap which will naturally complicat tracking.
+
+## Lattices
+
+A **lattice is the root structure holding the information about a
+``machine``. A machine may be as simple as a line of elements (like the elements of a Linac) or
+as complicated as an entire accelerator complex with multiple storage rings, Linacs, transfer
+lines, etc.
+
+Essentially, a **lattice**, has an array of **branches** with each branch describing part of the
+machine. Branches can be interconnected to form a unified whole.
+Branches can be interconnected using **Fork** elements. 
+This is used to simulate forking beam lines such as a connections to a transfer line, dump line, or an
+X-ray beam line. The **branch** from which other **branches** fork but is not forked to by any
+other branch is called a **root** branch.
+
+A lattice may contain multiple **root branches. For example, a pair of intersecting storage
+rings will generally have two root branches, one for each ring.
```

Comment:
I would suggest creating a definition block for all elements that appear more than twice in the lattice, and a lattice structure for the ordered elements.

Reply:
This will need development in future PRs